apiary.apib

```diff
@@ -690,6 +692,12 @@ If you receive a network timeout or other unexpected response (such as a 5xx) wh
 ## Funds [/funds]
 Real-time information on your account funding.
 
+a `POST` to the `/funds` endpoint allows you to add funds to your account. Some configuration must first be completed in the web application. You will need a credit card on file for the currencies you plan to fund. Next, take note of the funding fee and minimum charge amount. Both of those details can be found under `Purchase Credits` -> `Auto-charge`. The response to `POST` `/funds` is your updated account balances.
```

Comment:
Having `Purchase Credits` -> `Auto-charge` in back ticks makes it feel like code or part of the API.  I'd suggest taking those off and explicitly saying "Purchase Credits -> Auto-charge in the web application"